### PR TITLE
Fix audit gems failure

### PR DESCRIPTION
**Pull request by apoorv1316**

### CODE REVIEW

**Review by keshavbiswa — APPROVED** [review]

File: Gemfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 ruby "3.2.4"
 
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
-gem "rails", "~> 7.1.3"
+gem "rails", "~> 7.1.3.4"
```
Comment (keshavbiswa) [inline Gemfile:9]:
```suggestion
gem "rails", "~> 7.1.4"
```

**Review by keshavbiswa — CHANGES_REQUESTED** [review]

**Comment by keshavbiswa** [comment]
Tests are failing

**Comment by apoorv1316** [comment]
@keshavbiswa Please approve. I have fixed tests

**Review by keshavbiswa — APPROVED** [review]
LGTM!

**Pull request merged**